node/api.go

```diff
@@ -126,3 +127,21 @@ func (node *Node) GetConsensusViewChangingID() uint64 {
 func (node *Node) GetConsensusCurViewID() uint64 {
 	return node.Consensus.GetCurViewID()
 }
+
+// GetConsensusBlockNum returns the current block number of the consensus
+func (node *Node) GetConsensusBlockNum() uint64 {
+	return node.Consensus.GetBlockNum()
+}
+
+// GetConsensusInternal returns consensus internal data
+func (node *Node) GetConsensusInternal() rpc_common.ConsensusInternal {
+	return rpc_common.ConsensusInternal{
+		ViewID:        node.GetConsensusCurViewID(),
+		VCID:          node.GetConsensusViewChangingID(),
+		Mode:          node.GetConsensusMode(),
+		Phase:         node.GetConsensusPhase(),
+		BlockNum:      node.GetConsensusBlockNum(),
+		InSync:        !node.IsOutOfSync(node.Blockchain()),
```

Comment:
This locks the syncing service, prob best to not put this in the node metadata as its called quite frequently.

Reply:
Suggestion is to put it as a separate RPC.

Reply:
got it, will remove it.